src/installer/installer.go

```diff
@@ -304,8 +304,16 @@ func (i *installer) waitForController() error {
 		return err
 	}
 	predicate := func() bool {
-		return common.IsPodInStatus(kc, assistedControllerPrefix, assistedControllerNamespace,
+		controllerPod := common.GetPodInStatus(kc, assistedControllerPrefix, assistedControllerNamespace,
 			map[string]string{"job-name": assistedControllerPrefix}, v1.PodRunning, i.log)
+		if controllerPod != nil {
+			err = common.UploadPodLogs(kc, i.inventoryClient, i.ClusterID, controllerPod.Name, assistedControllerNamespace, common.ControllerLogsSecondsAgo, i.log)
```

Comment:
Unsure sending the logs here is the best option,
I think logs should be sent late as possible, right before restart.

Reply:
this is only controller logs, just to see what is going on in it before exit. Reboot will be set in ~ 5 seconds